Fix linting errors

```bash
../src/utils.ts
  146:1  error  This line has a length of 125. Maximum allowed is 120  max-len
  149:1  error  This line has a length of 121. Maximum allowed is 120  max-len
  152:1  error  This line has a length of 135. Maximum allowed is 120  max-len
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,10 @@
 # Changelog
 
-## `authmosphere 3.0.2`
+## `authmosphere 3.0.3`
+
+* fix linting error
+
+## `authmosphere 3.0.2 (unreleased)`
 
 * remove the last left over dependencies and files for TSLint
 * fix npm task

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "authmosphere",
-  "version": "3.0.2",
+  "version": "3.0.3",
   "description": "authmosphere is a library to support OAuth2 workflows in JavaScript projects.",
   "main": "./lib/src/index.js",
   "typings": "./lib/src/index.d.ts",

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -143,15 +143,18 @@ const rejectRequest: rejectRequest = (res, logger, status) => {
 const isCredentialsDirConfig = (options: Record<string | number | symbol, unknown>): options is CredentialsDirConfig =>
   options.credentialsDir !== undefined;
 
-const isCredentialsClientConfig = (options: Record<string | number | symbol, unknown>): options is CredentialsClientConfig =>
-  options.clientId !== undefined && options.clientSecret !== undefined;
+const isCredentialsClientConfig =
+  (options: Record<string | number | symbol, unknown>): options is CredentialsClientConfig =>
+    options.clientId !== undefined && options.clientSecret !== undefined;
 
-const isCredentialsUserConfig = (options: Record<string | number | symbol, unknown>): options is CredentialsUserConfig =>
-  options.applicationUsername !== undefined &&  options.applicationPassword !== undefined;
+const isCredentialsUserConfig =
+  (options: Record<string | number | symbol, unknown>): options is CredentialsUserConfig =>
+    options.applicationUsername !== undefined && options.applicationPassword !== undefined;
 
-const isPasswordGrantNoCredentialsDir = (options: Record<string | number | symbol, unknown>): options is CredentialsUserClientConfig =>
-  options.grantType === OAuthGrantType.PASSWORD_CREDENTIALS_GRANT &&
-   isCredentialsUserConfig(options) && isCredentialsClientConfig(options);
+const isPasswordGrantNoCredentialsDir =
+  (options: Record<string | number | symbol, unknown>): options is CredentialsUserClientConfig =>
+    options.grantType === OAuthGrantType.PASSWORD_CREDENTIALS_GRANT &&
+    isCredentialsUserConfig(options) && isCredentialsClientConfig(options);
 
 const checkCredentialsSource = (options: OAuthConfig) =>
   isCredentialsDirConfig(options) || isCredentialsClientConfig(options) || isPasswordGrantNoCredentialsDir(options);

